assets/src/blocks/Spreadsheet/Spreadsheet.js

```diff
@@ -89,20 +89,18 @@ export class Spreadsheet extends Component {
     );
   }
 
+  renderView() {
```

Comment:
Hmmm not sure I follow, this is not rendering the placeholder as we do in the frontend... maybe you saw it's not rendering, the reason the table is not being rendered is because of a bug I introduced in the endpoint here: https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/273/files#diff-765c584bd08bac4ddefd3a6bfc6e4706R126

I am validating the `sheet_id` as if it was an integer, I don't know why I thought so... just out of habit I guess...